src/db/migrate.rs

```diff
@@ -269,6 +269,19 @@ fn migrate_inner(version: Option<Version>, conn: &Connection, apply_mode: ApplyM
             // downgrade query
             "DROP TABLE blacklisted_crates;"
         ),
+        migration!(
+            context,
+            // version
+            7,
```

Comment:
```suggestion
            8,
```

Migration 7 is already taken by https://github.com/rust-lang/docs.rs/pull/567.

Reply:
This required a rebase.